### ISSUE TITLE
Support for Amazon Managed Workflows for Apache Airflow (MWAA)

### ISSUE THREAD

**Issue by vparv** [cloud-custodian/cloud-custodian#7979]
### Describe the feature

I am requesting support for Amazon Managed Workflows for Apache Airflow (MWAA). I'm using MWAA as I need to operate data pipelines by creating workflows and do not want to have to manage underlying infrastructure.

**Comment by PratMis** [comment]
I think it's already supported as of today.

**Comment by taabrcr** [comment]
Yes, but it is not working as expected, we always get an empty array of resources:

[INFO]	2024-03-12T13:02:21.542Z	d0246d04-1e07-4337-9bd9-be7e825650d7	Found resource ids:['arn:aws:airflow:<our-region>:<account-number>:environment/MyAirflowEnvironment']
[INFO]	2024-03-12T13:02:21.839Z	d0246d04-1e07-4337-9bd9-be7e825650d7	Resources []

Could you please check this? :)

**Comment by taabrcr** [comment]
We had to use "requestParameters.Name" instead of "responseElements.Arn", it would be nice to have this documented.
